src/logic/operate.js

```diff
@@ -0,0 +1,40 @@
+const Big = require('big.js');
+
+const operate = (ValueOne, ValueTwo, opt) => {
+  let result;
```

Comment:
- `valueOne` and `valueTwo` should be wrapped in Big class (you are only wrapping `valueOne`)

Reply:
I wrapped value 2 in big class